### DISCUSSION TITLE
Vue's transition-group breaks the new grid feature.

### DISCUSSION THREAD

**Discussion by antimanada** [tailwindlabs/tailwindcss#1509] [General]
Hi, im trying to use the new feature grid grid-cols-5... with vue transition-group but the result is rendered in just one column. I've inspected on devtools and i fount what us happening. The transition-group puto the hole v-for iteration inside a span and it endup in just one column.

Do you guys know how to fix it?

Here is the code:
![image](https://user-images.githubusercontent.com/60284418/77864115-40707d80-71fd-11ea-9992-7b48eb913c2c.png)

Here is the result:
![image](https://user-images.githubusercontent.com/60284418/77864138-626a0000-71fd-11ea-8a6b-ae2efe19e2ec.png)

Here is the devtools result:
![image](https://user-images.githubusercontent.com/60284418/77864212-bd9bf280-71fd-11ea-9a03-488e8d46912d.png)

**Comment by adamwathan** [comment]
The transition-group component creates an extra `span` wrapper element (which you can see in your dev tools screenshot) so your grid only has one direct child.

Try moving your grid classes directly onto the transition-group and adding the `tag` prop and setting it to `div`:

```html
<transition-group name="fade" tag="div" class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-8">
```

**Reply by antimanada** [reply]
It worked!

Thanks  very much!

**Comment by adamwathan** [comment]
Moving this to our discussions area, please reserve issues for actual Tailwind bugs or feature requests 👍🏻

**Reply by antimanada** [reply]
Ok!

Thankyou very much for your attention.  It worked.
